scripts/create-3scale-index.sh

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env bash
+#
+# Prereq:
+# - opm
+# - yq (4.24.2+)
+#
+# Function:
+# Script creates 3scale index/indexes for RHOAM 
+#
+# Usage:
+# REG=<REGISTRY> ORG=<QUAY ORG> IMAGE=<IMAGE NAME> BUILD_TOOL=<docker|podman>  VERSION=<VERSION TAG> ./scripts/create-3scale-index.sh
+# REG - registry of where to push the bundles and indexes, defaults to quay.io
+# ORG - organization of where to push the bundles and indexes
+# IMAGE - image name of the image to push
+# BUILD_TOOL - tool used for building the bundle and index, defaults to docker
+# VERSION - index version to build; this defines the max version and includes all bundles beneath it
+# TAG - version tag to be used for the image
+#
+# Example:
+# VERSION=0.8.1 ORG=acatterm ./scripts/create-3scale-index.sh
```

Comment:
hey! Not a big deal but do you think we could add a make command?

Reply:
Sure can do